tests/test_values.py

```diff
@@ -411,26 +410,23 @@ def test_email_url_value(self):
                 'EMAIL_HOST_PASSWORD': 'password',
                 'EMAIL_HOST_USER': 'user@domain.com',
                 'EMAIL_PORT': 587,
-                'EMAIL_USE_SSL': False,
                 'EMAIL_USE_TLS': True})
-        with env(EMAIL_URL='console://'):
+        with env(EMAIL_URL='consolemail://'):
```

Comment:
Is that change required?
Is it related to https://github.com/psgs/ConsoleMail?

Reply:
> Is it related to https://github.com/psgs/ConsoleMail?

No. I'm not aware of ConsoleMail. What I learned from django-environ is only `consolemail` is defined in email schemes, but no `console`. The `consolemail` maps to same backend `django.core.mail.backends.console.EmailBackend`.

Reply:
That's another behavior change then apparently AFAIK.

All those might be fine, and it's good to modernize, but might break setups/configs.

Reply:
Hi @blueyed, I added another commit to make `console://` still work in apps using it, and deprecate `console://`.